### ISSUE TITLE
What is the orientation accuracy in training set of phase-1?

### ISSUE THREAD

**Issue by Edwardmark** [Jack-CV/PCN#23]
Hi, 
Jack, I try to re-implement your PCN model, but I found that training phase1, the up-down face orientation prediction accuracy is very low(about 90% in training set). I think that is not well enough for the training of the second phase. Do I miss sth? Please comment, thank you.

Best, 
Edward

**Comment by Edwardmark** [comment]
@Jack-CV Please help me out. Did you use certain data balance trick or something like that to get 95% accuracy?

**Comment by JsBlueCat** [comment]
@Edwardmark  how about your box loss ?  the paper said the  ta tb tw but don't tell the ta* tb* tw*. Could you help me ?

**Comment by JsBlueCat** [comment]
@Edwardmark  thank you , but another question  : how to generate the training data? how to rotate them? Use guass distribution?

**Comment by Edwardmark** [comment]
@JsBlueCat Uniform distribution

**Comment by JsBlueCat** [comment]
@Edwardmark  how do you train the left-top point ? #10 in this issue   the author seems  define the cx and cy which is the center of the square rectangle.
but in his paper , he trains the PCN with ta tb tw ,which represents the left-top point of the bounding box and its width.  but after the rotation,the left-top point will change and hava some relations with rotate angle.

**Comment by Edwardmark** [comment]
@JsBlueCat just calculate the coordinates.

**Comment by Rogerluojie** [comment]
@Edwardmark 你好，请问你是怎样计算角度回归训练时的对应标签数据的？

**Comment by Edwardmark** [comment]
@Rogerluojie 根据左边变换计算

**Comment by Rogerluojie** [comment]
@Edwardmark 谢谢，可否列出具体的计算公式参考下，谢谢！！！

**Comment by Rogerluojie** [comment]
@Edwardmark 谢谢，可否列出具体的计算公式参考下，谢谢！！！

**Comment by Edwardmark** [comment]
@Rogerluojie https://www.zhihu.com/question/52027040
